src/testsuite/vec_int16_dummy.c

```diff
@@ -281,6 +159,8 @@ __test_mulinvuh (vui16_t n, const unsigned short M, const unsigned short a,
       q = vec_avg (q, n_1);
       if (s > 1)
 	result = vec_srhi (q, (s - 1));
+      else
+	result = q;
```

Comment:
indentation is off (looks like it was in the original code, too)

Reply:
looks like it is on you end.

I see 6 spaces vs 1 tab. 

This is after SDK format.